spec/index.bs

```diff
@@ -192,34 +192,82 @@ dictionary GPULimits {
 </script>
 
 
-Buffers {#buffers}
-==================
+# {{GPUBuffer}} # {#GPUBuffer}
 
-## GPUBuffer ## {#buffer}
+A {{GPUBuffer}} represents a block of memory that can be used in GPU operations.
+Data is stored in linear layout, meaning that each byte of the allocation can be
+addressed by its offset from the start of the {{GPUBuffer}}, subject to alignment
+restrictions depending on the operation..
 
-<script type=idl>
-interface GPUBuffer : GPUObjectBase {
-    Promise<ArrayBuffer> mapReadAsync();
-    Promise<ArrayBuffer> mapWriteAsync();
-    void unmap();
+{{GPUBuffer}} has the following internal slots:
 
-    void destroy();
-};
-</script>
+<dl dfn-type=attribute dfn-for="GPUBuffer">
+	: <dfn>\[[size]]</dfn>
```

Comment:
nit: instead of "which is an unsigned long long", this can match the formatting of IDL member attributes by doing something like [https://github.com/gpuweb/gpuweb/pull/422/commits/987d404632d3cf6d592db8e71aacfc23c3dbe54d#diff-599c3a4f36042dec5cdc606f3c30dc38R104](this):
```
    : <dfn>\[[size]]</dfn>, of type unsigned long long, readonly
```

Reply:
Did this but without the readonly annotation because it isn't clear when the internal slot becomes readonly. I'd rather have its readonlyness in a note.